pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>tech.jhipster.controlcenter</groupId>
     <artifactId>jhipster-control-center</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <version>0.1.0-SNAPSHOT</version>
```

Comment:
why do you change this?

Reply:
Because of a new feature, but I will change it back.

Reply:
Projection Version reverted!